aspnetcore/blazor/components/render-modes.md

```diff
@@ -105,110 +105,68 @@ To apply a render mode to a component instance use the [`@rendermode` Razor dire
 
 > [!NOTE]
 > Component authors should avoid coupling a component's implementation to a specific render mode. Instead, component authors should typically design components to support any render mode or hosting model. A component's implementation should avoid assumptions on where it's running (server or client) and should degrade gracefully when rendered statically. Specifying the render mode in the component definition may be needed if the component isn't instantiated directly (such as with a routable page component) or to specify a render mode for all component instances.
 
-> [!NOTE]
-> During .NET 8 *Release Candidate 2*, the Blazor Web App project template sets the interactive render mode with the [`@attribute` Razor directive](xref:mvc/views/razor#attribute) in template-generated sample components. The template will be updated to use the [`@rendermode` Razor directive](xref:mvc/views/razor#rendermode) for the release of .NET 8 in November.
->
-> Render mode             | Directive
-> ----------------------- | ---------
-> Interactive Server      | `@attribute [RenderModeInteractiveServer]`
-> Interactive WebAssembly | `@attribute [RenderModeInteractiveWebAssembly]`
-> Interactive Auto        | `@attribute [RenderModeInteractiveAuto]`
->
-> You can change the directives in an app generated by the project template to use the `@rendermode` directive.
-
 ## Prerendering
 
 Prerendering is enabled by default for interactive components.
 
-To disable prerendering for a component instance, pass the `prerender` flag with a value of `false` with either a new `InteractiveServerRenderMode` or `InteractiveAutoRenderMode`:
+<!-- UPDATE 8.0 Are there any simplifications for these at RTM? -->
 
-```razor
-<... @rendermode="new InteractiveServerRenderMode(prerender: false)" />
-```
+To disable prerendering for a component instance, pass the `prerender` flag with a value of `false` to the render mode:
 
-```razor
-<... @rendermode="new InteractiveAutoRenderMode(prerender: false)" />
-```
+* `<... @rendermode="new InteractiveServerRenderMode(prerender: false)" />`
+* `<... @rendermode="new InteractiveWebAssemblyRenderMode(prerender: false)" />`
+* `<... @rendermode="new InteractiveAutoRenderMode(prerender: false)" />`
 
-To disable prerendering in a component definition, pass the `prerender` flag with a value of `false` with either the `RenderModeInteractiveServer` attribute or `RenderModeInteractiveAuto` attribute:
+To disable prerendering in a component definition, pass the `prerender` flag with a value of `false` to the render mode:
 
-```razor
-@page "..."
-@attribute [RenderModeInteractiveServer(prerender: false)]
-```
+<!-- UPDATE 8.0 Should we be doing this the way that the PU
```

Comment:
@guardrex do you have another option for this? I'm just trying to understand the question here.

Reply:
Ok, I think I see what you meant here: use the `@rendermode` based syntax rather than the `@attribute` based one. I think using the @rendermode is preferrable.

Reply:
I'll update this on the next commit.